test/Interop/Cxx/class/constructors-copy-irgen.swift

```diff
@@ -4,6 +4,9 @@
 // RUN: %swift -module-name Swift -target armv7-none-linux-androideabi -dump-clang-diagnostics -I %S/Inputs -enable-experimental-cxx-interop -emit-ir %s -parse-stdlib -parse-as-library -disable-legacy-type-info | %FileCheck %s -check-prefix=ITANIUM_ARM
 // RUN: %swift -module-name Swift -target x86_64-unknown-windows-msvc -dump-clang-diagnostics -I %S/Inputs -enable-experimental-cxx-interop -emit-ir %s -parse-stdlib -parse-as-library -disable-legacy-type-info | %FileCheck %s -check-prefix=MICROSOFT_X64
 
+// REQUIRES: CODEGENERATOR=X86
+// REQUIRES: CODEGENERATOR=ARM
```

Comment:
This should not be required. These test should work on any platform. Were you seeing failures?

Reply:
Locally. I don't normally build with the ARM generator, but that means that the `armv7-none-linux-androideabi` run fails because swift can't compile for it.

Reply:
(and of course targeting x86_64 requires the X86 backend be enabled)